pandas/core/generic.py

```diff
@@ -6730,9 +6730,9 @@ def fillna(
             each index (for a Series) or column (for a DataFrame).  Values not
             in the dict/Series/DataFrame will not be filled. This value cannot
             be a list.
-        method : {{'backfill', 'bfill', 'pad', 'ffill', None}}, default None
```

Comment:
pad is deprecated, so I removed from the fillna docstrings. backfill is deprecated in groupby but not DataFrame or Series. I'll look into this for a followup.